users/README.md

```diff
@@ -1,27 +1,87 @@
 # Users
 
+```json
+{
+	"id": "string",
+	"username": "string",
+	"first_name": "string",
+	"last_name": "string",
+	"email": "string",
+	"phone": "number",
+	"yoe": "number",
+	"company": "string",
+	"designation": "string",
+	"img": "string",
```

Comment:
I am not getting what is the img is used for.

Reply:
This was already in the document, so will address this in later PR if needed.